### DISCUSSION TITLE
What do you use to host?

### DISCUSSION THREAD

**Discussion by TisMeCastle** [SudhanPlayz/Discord-MusicBot#159] [Q&A] [answered]
I tried music bots but only the opus ones worked. Yet Heroku doesn't support any opus for some reason. So my bot cant have music.

**Comment by ItzSudhan** [comment, marked as answer]
Use repl its fine with me

**Comment by ddodogames** [comment]
i use glitch so yep

**Comment by KainesLaw** [comment]
u can use repl, and to keep the bot online, use https://uptimerobot.com/dashboard.php#mainDashboard
